apps/rig_inbound_gateway/test/rig/api_proxy/router_test.exs

```diff
@@ -174,6 +174,17 @@ defmodule RigInboundGateway.ApiProxy.RouterTest do
     assert conn.resp_body =~ "{\"response\": \"file uploaded successfully\"}"
   end
 
+  test_with_server "Any request should include forward headers", @env do
+    route("/myapi/direct", Response.ok!(~s<{"status":"ok"}>))
+
+    conn = call(Router, build_conn(:get, "/myapi/direct"))
+
+    assert conn.status == 200
+    assert conn.resp_body =~ "{\"status\":\"ok\"}"
+    assert [forwarded_header] = get_req_header(conn, "Forwarded")
```

Comment:
Fails:
```
  1) test Any request should include forward headers (RigInboundGateway.ApiProxy.RouterTest)
     test/rig/api_proxy/router_test.exs:177
     match (=) failed
     code:  assert [forwarded_header] = get_req_header(conn, "Forwarded")
     right: []
     stacktrace:
       test/rig/api_proxy/router_test.exs:184: (test)
```

Reply:
"forwarded" was changed to small letter. missed to reflect this in the test..